src/qt/applicationbase/inviwoapplicationqt.cpp

```diff
@@ -135,19 +135,20 @@ void FileSystemObserverQt::fileChanged(QString fileName) {
 
 }  // namespace
 
-InviwoApplicationQt::InviwoApplicationQt(int& argc, char** argv, const std::string& displayName)
+InviwoApplicationQt::InviwoApplicationQt(int& argc, char** argv, std::string displayName,
+                                         std::string organizationName,
+                                         std::string organizationDomain)
     : QApplication(argc, argv)
     , InviwoApplication(argc, argv, displayName)
     , mainWindow_(nullptr)
     , uiLocal_(getCurrentStdLocale())
     , undoTrigger_{[]() {}} {
 
     setAttribute(Qt::AA_NativeWindows);
-    setWindowIcon(QIcon(":/inviwo/inviwo_light.png"));
```

Comment:
does the default CMake file set the appropriate window icon?

Reply:
Yes, we set the following:

```
set(RES_FILES "")
if(WIN32)
    set(RES_FILES ${RES_FILES} "${IVW_RESOURCES_DIR}/inviwo.rc")
elseif(APPLE)
    set_source_files_properties(${IVW_ROOT_DIR}/resources/inviwo/inviwo_light.icns PROPERTIES MACOSX_PACKAGE_LOCATION Resources)
    set(RES_FILES ${RES_FILES} ${IVW_ROOT_DIR}/resources/inviwo/inviwo_light.icns)
endif()
```

I have not tested this on windows and Linux.

Reply:
Tested on Windows and there it seems like it is necessary to call setWindowIcon. So, I added an option to specify the icon